tests/zfs-tests/tests/functional/cli_root/zfs_send/zfs_send_holds.ksh

```diff
@@ -0,0 +1,176 @@
+#!/bin/ksh -p
+#
+# CDDL HEADER START
```

Comment:
IIRC there is a newer version of the CDDL header, maybe we should use that instead.